client/state/action-types.js

```diff
@@ -486,9 +486,9 @@ export const THEME_CLEAR_ACTIVATED = 'THEME_CLEAR_ACTIVATED';
 export const THEME_DETAILS_REQUEST = 'THEME_DETAILS_REQUEST';
 export const THEME_DETAILS_RECEIVE = 'THEME_DETAILS_RECEIVE';
 export const THEME_DETAILS_RECEIVE_FAILURE = 'THEME_DETAILS_RECEIVE_FAILURE';
-export const THEME_RECEIVE_CURRENT = 'THEMES_RECEIVE_CURRENT';
-export const THEME_REQUEST_CURRENT = 'THEME_REQUEST_CURRENT';
-export const THEME_REQUEST_CURRENT_FAILURE = 'THEME_REQUEST_CURRENT_FAILURE';
+export const THEME_CURRENT_REQUEST_SUCCESS = 'THEME_CURRENT_REQUEST_SUCCESS';
```

Comment:
Nomenclature is hard 😄. This sounds a bit like it's the _current request_ for a theme, not a request for the  _current theme_. Also, to align with the new action name, it should be _active_, not _current_.

Soo... maybe `ACTIVE_THEME_REQUEST` etc? (I think it's fine to not prefix with `THEME_`.)